src/Models/Country.php

```diff
@@ -31,6 +31,14 @@ protected static function boot()
         });
     }
 
+    public static function fromAbbreviation(string $abbreviation): self
+    {
+        /** @var Country $result */
+        $result = static::query()->where('abbreviation', '=', $abbreviation)->firstOrFail();
+
+        return $result;
+    }
```

Comment:
This is a great feature!

Reply:
There should probably be similar in State, Location, Timezone, etc -- basically, anywhere there is a unique short string that is typically well known or commonly used.